odoo_office365_contacts==UPDATED/odoo_office365_contacts/models/office365lib.py

```
@@ -0,0 +1,44 @@
+            headers={
+                "Authorization": self.Session.token,
+                "Content-Type": "application/json",
+            },
+        )
+        if request_type == "DELETE":
+            return conn.getresponse()
+        response = json.loads(conn.getresponse().read())
+        return json.dumps(response)
+
+
+class Office365:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)